subsys/bluetooth/controller/ll_sw/openisa/hal/RV32M1/ecb.c

```diff
@@ -8,16 +8,20 @@
 
 #include <string.h>
 
+#include <zephyr/bluetooth/hci.h>
+
 #include <zephyr/sys/dlist.h>
 #include <zephyr/sys/byteorder.h>
 
 #include "hal/ecb.h"
 
-#define LOG_MODULE_NAME bt_ctlr_rv32m1_ecb
-#include "common/log.h"
+#include <zephyr/logging/log.h>
+
 #include "hal/debug.h"
 #include "fsl_cau3_ble.h"
 
+LOG_MODULE_REGISTER(bt_ctlr_rv32m1_ecb, LOG_LEVEL_DBG);
```

Comment:
shouldn't here be a level from kconfig?

Reply:
Not really if we look at how it was done previously. There was no `BT_DBG_ENABLED` so the default behavior was to use `LOG_LEVEL_DBG`. As it's a refactor PR I don't think that I should change that behavior.

You can check by yourself with that:
```c
#define XSTR(x) STR(x)
#define STR(x) #x
#pragma message "Log level: " XSTR(LOG_LEVEL)
```